Program.cs

```diff
@@ -2,60 +2,110 @@
 
 namespace objected_oriented_programming
 {
-   abstract class MOSGORGEOTREST
+    enum typesOfBankAccounts
     {
-        string ROLE { get; set; }
-        public abstract void Act();
+        checking_account,
+        savings_account,
+        investment_account
     }
 
-    class CLient : MOSGORGEOTREST
+    class BankAccount
```

Comment:
Модификатор доступа потеряли
```suggestion
    internal class BankAccount
```